powersimdata/scenario/delete.py

```diff
@@ -41,25 +40,21 @@ def delete_scenario(self, confirm=True):
         self._scenario_list_manager.delete_entry(scenario_id)
         self._execute_list_manager.delete_entry(scenario_id)
 
-        wildcard = f"{scenario_id}_*"
-
         print("--> Deleting scenario input data on server")
```

Comment:
I believe we can delete locally and hence just print "Deleting scenario input data"